### PR TITLE
Add a 'How to join Slack' FAQ entry

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: content/help/faq/general/how-to-join-slack.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: How can I join Istio Slack workspace?
+weight: 180
+---
+
+Istio Slack is restricted to Istio Developers. See the [Community Roles](https://github.com/istio/community/blob/master/ROLES.md)
+guide, specifically the **Member** role. In this page you can find instructions on how to apply for Istio membership,
+if you have contributed to the project, and thus get access to Istio Slack workspace.
```
Comment (linsun) [inline content/help/faq/general/how-to-join-slack.md]: if you have contributed to the project... -> If your membership request has been approved, You will be invited to join the Istio Slack.

File: content/help/faq/general/how-to-join-slack.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: How can I join Istio Slack workspace?
+weight: 180
+---
+
+Istio Slack is restricted to Istio Developers. See the [Community Roles](https://github.com/istio/community/blob/master/ROLES.md)
+guide, specifically the **Member** role. In this page you can find instructions on how to apply for Istio membership,
```
Comment (linsun) [inline content/help/faq/general/how-to-join-slack.md]: could you link to the member directly like:  https://github.com/istio/community/blob/master/ROLES.md#member

File: i18n/en.toml [hunk 3]
```diff
@@ -29,7 +29,7 @@ other = "Access our team drive if you'd like to take a look at the Istio technic
 other = "If you'd like to contribute to the Istio project, consider participating in our working groups"
 
 [slack_desc]
-other = "Interactively discuss development issues with the Istio community on Slack (invitation-only)"
+other = "Interactively discuss development issues with the Istio community on Slack (Developers only, see FAQ)"
```
Comment (linsun) [inline i18n/en.toml]: could we link to the FAQ?

Comment (jwendell) [reply i18n/en.toml]: Unfortunately not, as this is a HTML tooltip (that appears at footpage).

**Comment by jwendell** [comment]
@linsun yep, people ask all the time about that. Having it in FAQ hopefully improves a little bit the situation.
I've done the changes you requested, thanks.

File: i18n/en.toml (hunk 3 above)
Comment (linsun) [reply i18n/en.toml]: got it, thanks!

**Pull request merged**